src/rules/order.js

```diff
@@ -244,14 +244,19 @@ function getSorter(ascending) {
   const multiplier = ascending ? 1 : -1;
 
   return function importsSorter(importA, importB) {
-    let result;
-
-    if (importA < importB) {
+    let result = 0;
+
+    const A = importA.split('/');
+    const B = importB.split('/');
+    const a = A.length;
+    const b = B.length;
+    
+    if (A < B) {
       result = -1;
-    } else if (importA > importB) {
+    } else if (A > B) {
       result = 1;
-    } else {
-      result = 0;
+    } else if (a !== b) {
+      result = a < b ? -1 : 1;
```

Comment:
codecov says this line isn't covered by tests; can we find a test case that covers it?

Reply:
Looks like we can get rid of that line of code as `<` operator takes care of length as well.